nibabel/gifti/gifti.py

```diff
@@ -680,6 +680,45 @@ def get_arrays_from_intent(self, intent):
         it = intent_codes.code[intent]
         return [x for x in self.darrays if x.intent == it]
 
+    def agg_data(self, intent_code=None):
+        """
+        Retrun a numpy arrary of aggregated GiftiDataArray of the same intent code
+        or
+        Retrun GiftiDataArray in tuples for surface files
+
+        Parameters
+        ----------
+        intent_code : None, string, integer or tuple of string, optional
+            Intent code, or string describing code.
+            Accept tuple that contains multiple intents to specify the order.
+
+        Returns
+        -------
+        all_data : tuple of ndarray or ndarray
```

Comment:
Return values aren't named.

```suggestion
        tuple of ndarrays or ndarray
```